gcd/src/test/resources/datastore-indexes.xml

```diff
@@ -0,0 +1,27 @@
+<!--
+  ~ Copyright 2017, TeamDev Ltd. All rights reserved.
```

Comment:
Why do we insert `~` at the beginning of each line? Is this a part of the project copyright profile?